index.html

```diff
@@ -1,40 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-  </head>
-
-  <body>
-    <header id="header">
-      <div class="logo">
-        <img id="header-img" src="https://w7.pngwing.com/pngs/572/134/png-transparent-trumpet-musical-instrument-icon-cartoon-black-trumpet-text-logo-brass-instrument.png">
```

Comment:
@codedbystuart , I suggest that you provide an `alt` attribute to the image, just a simple description for the image to be displayed.